packages/data-context/test/unit/data/ProjectConfigIpc.spec.ts

```diff
@@ -38,9 +38,10 @@ describe('ProjectConfigIpc', () => {
   context('forkChildProcess', () => {
     // some of these node versions may not exist, but we want to verify
     // the experimental flags are correctly disabled for future versions
-    const NODE_VERSIONS = ['18.20.4', '20.17.0', '22.7.0', '22.11.4', '22.12.0', '22.15.0']
+    const NODE_VERSIONS = ['18.20.4', '20.17.0', '20.19.0', '21.0.0', '22.7.0', '22.11.4', '22.12.0', '22.15.0']
```

Comment:
Might be better to write this test Node 22.0.0 does not apply the `--no-experimental-require-module` and `--no-experimental-detect-module` options since Cypress technically does not support Node 21
```suggestion
    const NODE_VERSIONS = ['18.20.4', '20.17.0', '20.19.0', '22.0.0', '22.7.0', '22.11.4', '22.12.0', '22.15.0']
```